nvtabular/framework_utils/torch/utils.py

```diff
@@ -113,20 +113,21 @@ def process_epoch(
     with torch.set_grad_enabled(train):
         y_list, y_pred_list = [], []
         for idx, batch in enumerate(iter(dataloader)):
-            x_cat, x_cont, y = transform(batch)
-            if device:
-                x_cat = x_cat.to(device)
-                x_cont = x_cont.to(device)
-                y = y.to(device)
+            x, y = batch
+            # x_cat, x_cont, y = transform(batch)
+            # if device:
+            #     x_cat = x_cat.to(device)
+            #     x_cont = x_cont.to(device)
+            #     y = y.to(device)
```

Comment:
Does this commented code need to be removed to make this work? Does it still work if it's uncommented?